.coveragerc

```diff
@@ -145,6 +145,7 @@ omit =
     homeassistant/components/cppm_tracker/device_tracker.py
     homeassistant/components/cpuspeed/sensor.py
     homeassistant/components/crimereports/sensor.py
+    homeassistant/components/crownstone/*
```

Comment:
Please do not exclude `config_flow.py` from the test coverage.

We require a configuration flow to have 100% test coverage. Currently, there are no tests in this PR, so that needs to be added.

Reply:
Thanks for the review, i'll update the PR with the config flow test